Calculator/ViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Oleksii Kolomiiets on 4/4/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        inTheMiddleOfTyping = false
+    }
+    
+    @IBOutlet weak var display: UILabel!
+    
+    private var inTheMiddleOfTyping: Bool!
```

Comment:
What is the reason of "forced optional"? Can you avoid it?

Reply:
There are no reason. I will give it default value 'false'.